integration/env/common.go

```diff
@@ -44,3 +53,7 @@ func CircleSHA() string {
 func KeepResources() string {
 	return keepResources
 }
+
+func RegistryPullSecret() string {
+	return registryPullSecret
+}
```

Comment:
DO we have the explicit pull secret magic in the other operators as well? Why do we need this here now? Wasn't this magically done somewhere else? How do you want to roll with this?

Reply:
Most operators have it, you didn't see it before because it was in some hidden env variable before. See `aws-operator` which does it the same way I did:
https://github.com/giantswarm/e2etemplates/blob/master/pkg/chartvalues/aws_operator.go#L17

And `azure-operator` which has it still in some hidden env variable:
https://github.com/giantswarm/azure-operator/blob/master/integration/template/azure_operator_charts_values.go#L46

So I would say that it's progress and we atleast have some vision on it.